rootfs/etc/nginx/lua/balancer/sticky.lua

```diff
@@ -73,7 +74,17 @@ function _M.balance(self)
   if not key then
     local random_str = string.format("%s.%s", ngx.now(), ngx.worker.pid())
     key = encrypted_endpoint_string(self, random_str)
-    set_cookie(self, key)
```

Comment:
Why are you taking this out of the `if` condition? The idea was to set cookie _only_ when it does not exist.